src/templates/viewer-html-template.ts

```diff
@@ -238,7 +234,15 @@ const template = /* html */ `
                     }
                 }
 
-                entity.script.create(FrameScene);
+                const appElement = await document.querySelector('pc-app').ready();
+                const cameraElement = await document.querySelector('pc-entity[name="camera"]').ready();
+
+                const app = await appElement.app;
+                const camera = cameraElement.entity;
+
+                camera.camera.clearColor = new Color(viewerSettings.background.color);
```

Comment:
You should be able to do just:

```
cameraElement.clearColor =
cameraElement.fov =
```

Reply:
the next line sets .script on the same object, which isn't supported by camera web element.